frontend/src/components/SecretKeyVerificationInput/SecretKeyVerificationInput.tsx

```diff
@@ -0,0 +1,169 @@
+import { useCallback, useMemo, useRef } from 'react'
+import { RegisterOptions, useForm } from 'react-hook-form'
+import { BiUpload } from 'react-icons/bi'
+import {
+  FormControl,
+  FormErrorMessage,
+  IconButton,
+  Input,
+  Skeleton,
+  Stack,
+} from '@chakra-ui/react'
+
+import { GUIDE_SECRET_KEY_LOSS } from '~constants/links'
+import { useIsMobile } from '~hooks/useIsMobile'
+import formsgSdk from '~utils/formSdk'
+
+import Button from '../Button'
+import FormLabel from '../FormControl/FormLabel'
+import Link from '../Link'
+
+const SECRET_KEY_NAME = 'secretKey'
+const SECRET_KEY_REGEX = /^[a-zA-Z0-9/+]+={0,2}$/
```

Comment:
/nit should refer to this as base64 regex instead as we're checking if this is a base64

Reply:
yeah technically it's a base64 regex, haha. didn't modify anything cuz it was just a refactor

in the future we may want to refactor again to combine `SecretKeyVerificationInput` and `SecretKeyActivationModal`